core/src/main/scala/cats/syntax/alternative.scala

```diff
@@ -13,11 +13,20 @@ trait AlternativeSyntax {
     new GuardOps(b)
 }
 
-final class UniteOps[F[_], G[_], A](private val fga: F[G[A]]) extends AnyVal {
+final class UniteOps[F[_], G[_], A] private[syntax] (protected val fga: F[G[A]])
+    extends AnyVal
+    with UniteOpsBinCompat0[F, G, A] {
+
+  @deprecated("use a FlatMap-constrained instead", "2.6.2")
```

Comment:
For consistency with your other deprecation messages:

```suggestion
  @deprecated("use a FlatMap-constrained version instead", "2.6.2")
```

Reply:
Fixed.